datasets/peer_read/peer_read.py

```diff
@@ -0,0 +1,306 @@
+                        }
+                        for reference_mention in metadata["referenceMentions"]
+                    ]
+
+                    yield id_, {
+                        "name": data.get("name", ""),
+                        "metadata": metadata,
+                    }
+                elif self.config.name == "reviews":
+                    yield id_, {
+                        "id": str(data.get("id", "")),
+                        "conference": str(data.get("conference", "")),
+                        "comments": str(data.get("comments", "")),
+                        "subjects": str(data.get("subjects", "")),
+                        "version": str(data.get("version", "")),
+                        "date_of_submission": str(data.get("date_of_submission", "")),
+                        "title": str(data.get("title", "")),
+                        "authors": data.get("authors", [])
+                        if isinstance(data.get("authors"), list)
+                        else [data.get("authors", "")],
+                        "accepted": str(data.get("accepted", "")),
+                        "abstract": str(data.get("abstract", "")),
+                        "histories": self._parse_histories(data.get("histories", [])),
+                        "reviews": [
+                            {
+                                "date": str(review.get("date", "")),
+                                "title": str(review.get("title", "")),
+                                "other_keys": str(review.get("other_keys", "")),
+                                "originality": str(review.get("originality", "")),
+                                "comments": str(review.get("comments", "")),
+                                "is_meta_review": str(review.get("is_meta_review", "")),
+                                "is_annotated": str(review.get("is_annotated", "")),
+                                "recommendation": str(review.get("recommendation", "")),
+                                "replicability": str(review.get("replicability", "")),
+                                "presentation_format": str(review.get("presentation_format", "")),
+                                "clarity": str(review.get("clarity", "")),
+                                "meaningful_comparison": str(review.get("meaningful_comparison", "")),
+                                "substance": str(review.get("substance", "")),
+                                "reviewer_confidence": str(review.get("reviewer_confidence", "")),
+                                "soundness_correctness": str(review.get("soundness_correctness", "")),
+                                "appropriateness": str(review.get("appropriateness", "")),
+                                "impact": str(review.get("impact")),
+                            }
+                            if isinstance(review, dict)
+                            else {}
```

Comment:
Here you allow to add empty reviews, is this expected ?

Reply:
yes some of the papers don't have reviews

Reply:
For a paper that doesn't have reviews wouldn't it be better to have `"reviews": []` instead of a list with empty dictionaries in it ? Indeed here you may end up with `"reviews": [{}]`

Reply:
Agreed :) Corrected it. @lhoestq